src/storage/v2/storage.cpp

```diff
@@ -529,6 +538,7 @@ Result<std::vector<VertexAccessor>> Storage::Accessor::TryDeleteVertices(const s
 }
 
 void Storage::Accessor::MarkEdgeAsDeleted(Edge *edge) {
+  OOMExceptionEnabler oom_exception;
```

Comment:
I think exception enablers in this file aren't necessary if delete operation in the operator tree has exceptions enabled.

Reply:
and I think you added to the DeleteCursor